2-mandatory/7-eligible-students.js

```diff
@@ -13,11 +13,18 @@
     (see tests to confirm how this data will be structured)
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
-
-function eligibleStudents(attendances) {}
+// another solution
+//  function eligibleStudents(attendances) {
+//    let eligibleStudentsList = [];
+//    attendances
+//      .filter((student) => student.attendance >= 8)
+//      .map((student) => eligibleStudentsList.push(student.name));
+//    console.log(eligibleStudentsList);
+//    return eligibleStudentsList;
+//  }
```

Comment:
Make sure your codes line 16 - 24 are not commented out, otherwise it wont run and work.